### PR TITLE
:closed_lock_with_key: Refine security headers

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
I've added and configured helmet https://helmetjs.github.io/ for the same settings as in beta health (with some modifications to the content security whitelisting).

Redirection of http traffic to https is done at the infrastructure level so I have not implemented 

``` app.use(enforce.HTTPS({ ...}));```

enforce.HTTPS caused a large number of tests to fail and so, if it is felt to be required, this will need resolving.

I have, however, implemented [Strict-Transport-Security](https://helmetjs.github.io/docs/hsts/) as this makes the redirection to HTTPS more secure. See [here](https://scotthelme.co.uk/hsts-the-missing-link-in-tls/) for the details.

File: config/express.js [hunk 1]
```diff
@@ -21,7 +21,64 @@ module.exports = (app, config) => {
...
+        '\'self\'',
+        'data:',
+        'www.google-analytics.com',
+        'statse.webtrendslive.com',
+        'hm.webtrends.com',
+      ],
+      styleSrc: [
+        '\'self\'',
+        'fast.fonts.net',
+        'https://dhrlmnmyf2njb.cloudfront.net/',
```
Comment (neilbmclaughlin) [inline config/express.js]: Note: This URL is hard coded in the CSS too. Have raised an issue to make it configurable through an env var.

**Comment by st3v3nhunt** [comment]
@neilbmclaughlin PR #170 Fixes a couple of errors with inline scripts/styles.

There is an issue with Firefox on my machine - the page isn't rendering full screen but it isn't on the beta domain either. I'll raise a separate issue for that.

**Comment by neilbmclaughlin** [comment]
Added commit 4fdab81 :heavy_check_mark: Prevent inline script/style execution refusal
Removed commit 7f3db3e :no_bicycles: Add Strict-Transport-Security header

**Comment by neilbmclaughlin** [comment]
Ready for approval and merge

**Comment by st3v3nhunt** [comment]
As agreed with Adam (not in the GitHub org?!?) the `Strict-Transport-Security` header has been removed.

The rating is now a B https://securityheaders.io/?q=https%3A%2F%2Fconnecting-to-services-pr-167.herokuapp.com%2Ffinders%2Ffind-help&hide=on&followRedirects=on

Issue #172 has been raised to tackle inline script content policy

**Comment by st3v3nhunt** [comment]
[Add http security headers](https://trello.com/c/6aJCdJP6/561-add-http-security-headers)

**Pull request merged**